modules/meta.analysis/R/meta.analysis.summary.R

```diff
@@ -36,7 +36,7 @@ pecan.ma.summary <- function(mcmc.object, pft, outdir, threshold = 1.2, gg = FAL
   for (trait in names(mcmc.object)) {
     
     if (gg) {
-      gg <- require(ggmcmc)
+      gg <- requireNamespace(ggmcmc)
```

Comment:
```suggestion
      gg <- requireNamespace("ggmcmc")
```
Also, this sort of thing is usually included in the part of the function that needs the package (i.e. line 42).  Typically something like:

```r
if(gg) {
    if(!requireNamespace("ggmcmc")) {
      stop("`gg=TRUE` requires the 'ggmcmc' package. Please install 'ggmcmc' or run with `gg=FALSE`")
    } else {
      <code that should run when gg = TRUE>
    }
  }
```

Reply:
Good catch, @Aariq! Looks like we were reviewing at the same time -- I'm marking yours resolved since I incorporated it into one of my suggestions below.